cockpit/en.json

```
@@ -358,8 +359,18 @@
     "VARIABLE_SERIALIZED_VALUE": "Serialized value",
     "TIME_TO_LIVE_DAY": "{{ttl}} day",
     "TIME_TO_LIVE_DAYS": "{{ttl}} days",
-    "TIME_TO_LIVE_MESSAGE_ERR": "Failed to update history time to live",
+    "TIME_TO_LIVE_MESSAGE_ERR": "Failed to update history time to live.",
     "TIME_TO_LIVE_NULL": "null",
+    "TIME_TO_LIVE_POPUP_TITLE": "Edit History Time To Live value",
+    "TIME_TO_LIVE_POPUP_SUBTITLE": "Select one of the following options:",
+    "TIME_TO_LIVE_POPUP_RADIO_UPDATE": "Update Value",
+    "TIME_TO_LIVE_POPUP_RADIO_UPDATE_TOOLTIP": "Updates the value with the provided one",
+    "TIME_TO_LIVE_POPUP_RADIO_CLEAR": "Clear Value",
+    "TIME_TO_LIVE_POPUP_RADIO_CLEAR_TOOLTIP": "Sets the value to null",
+    "TIME_TO_LIVE_POPUP_STATE_STATUS": "Finished",
+    "TIME_TO_LIVE_POPUP_STATE_SUCCESS": "Updated the history time to live value.",
+    "TIME_TO_LIVE_POPUP_STATE_SUCCESS_MESSAGE_UPDATE": "The history time to live value has been updated to '{{ttl}}' days successfully.",
```

We are not consistent in our codebase if we quote values or not. However, we use double quotes if we to (e.g. https://github.com/camunda/camunda-webapp-translations/blob/c61217f42ea139f0cec5ac87160349c34e446623/cockpit/en.json#L130).

We can also remove the quotes completely to be consistent with line `TIME_TO_LIVE_POPUP_RADIO_CLEAR_TOOLTIP`:
```suggestion
    "TIME_TO_LIVE_POPUP_STATE_SUCCESS_MESSAGE_UPDATE": "The history time to live value has been updated to {{ttl}} days successfully.",
```

---

I removed them completely from ttl texts.